charts/astronomer/values.yaml

```diff
@@ -15,19 +15,19 @@ tolerations: []
 images:
   commander:
     repository: astronomerinc/ap-commander
-    tag: 0.18.0
+    tag: 0.21.0
     pullPolicy: IfNotPresent
   registry:
     repository: astronomerinc/ap-registry
     tag: 3.11.5-4
     pullPolicy: IfNotPresent
   houston:
-    repository: astronomerinc/ap-houston-api
-    tag: 0.20.9
+    repository: asoldatenko/ap-houston-api
+    tag: 0.21.3
```

Comment:
need wait for houston-api image:
https://app.circleci.com/pipelines/github/astronomer/houston-api/2170/workflows/270de444-9c90-4138-85b8-316eb283e19a/jobs/7300